system/lib/compiler-rt/lib/lsan/lsan.cc

```diff
@@ -59,7 +68,16 @@ static void InitializeFlags() {
   // Override from user-specified string.
   const char *lsan_default_options = MaybeCallLsanDefaultOptions();
   parser.ParseString(lsan_default_options);
+#if SANITIZER_EMSCRIPTEN
+  char *options = (char*) EM_ASM_INT({
+    return allocateUTF8(Module['LSAN_OPTIONS'] || 0,
```

Comment:
does speed matter here? EM_JS is faster.

Reply:
I think this is fine. This is run exactly once on startup, so speed doesn't really matter.